kernel_gateway/gatewayapp.py

```diff
@@ -312,19 +320,23 @@ def init_configurables(self):
                     self.prespawn_count, self.max_kernels)
                 )
 
-        if self.api == 'notebook-http':
-            self.kernel_pool = ManagedKernelPool(
-                self.prespawn_count,
-                self.kernel_manager
-            )
-        else:
-            self.kernel_pool = KernelPool(
-                self.prespawn_count,
-                self.kernel_manager
-            )
+        
+        api_module = self._load_api_module(self.api)
+        func = getattr(api_module, 'create_personality', None)
```

Comment:
It's the default return value if not found, not an arg being passed during invocation. Like the possible ImportError above, it's simpler to let it fail as an AttributeError with the reported line showing what was not found than to intentionally do anything more elaborate...right?